### ISSUE TITLE
Glyph sizing

### ISSUE THREAD

**Issue by kentiskis** [EtheaDev/StyledComponents#12]
hi,

in case when there are several StyledNavigators on form, then glyphs of last one are different (smaller).

Attaching picture and project source.

I'm running Delphi 11.3.
Custom Design PPI is 150.
Display Scale is 150 also.

Not sure if it is issue of StyledNav or not. 
Any thoughts are welcome.
Thanks,

Antanas
[demo.zip](https://github.com/EtheaDev/StyledComponents/files/14318873/demo.zip)
![pic1](https://github.com/EtheaDev/StyledComponents/assets/160327556/89be4356-1b56-4639-8ab6-eda14bfd3cba)

**Comment by carloBarazzetta** [comment]
It's a strange problem caused by pixelsperinch = 150.
Loading your dfm in a "96 dpi" designer the problem disappear...
I'll investigate...

**Comment by carloBarazzetta** [comment]
Please try with this fix (then recompile the packages) and give me a feed-back:
```
procedure TStyledDBNavigator.ChangeScale(M, D: Integer; isDpiChange: Boolean);
begin
  inherited;
  if isDpiChange and (M <> D) then
  begin
    ProcessButtons(
      procedure (ABtn: TStyledNavButton)
      begin
        ABtn.ChangeScale(M, D);
      end
    );
  end;
end;

```

**Comment by kentiskis** [comment]
Yes, this fix solved the issue. Thank you for so fast and effective support!

**Comment by carloBarazzetta** [comment]
Fixed in 3.2.1 version
